spec/requests/code_owner_projects.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Code Owner Projects' do
+  describe '#index' do
+    let(:user) { create(:user) }
+    subject { get "/users/#{user.id}/projects" }
+    context 'when a user has not projects as code owner' do
+      before do
+        subject
+      end
```

Comment:
I'm not too fond on doing the _action_ section of the test in the `before`.
And in this case, because of the context `'when projects has projects as code owner'`, isn't it executing it twice for those tests?
I'm open to anyone's thoughts about this!

Reply:
the subject just stores the request in itself but is not executing it, so the it is executed `before` every it in the context that it is declared.

On the other hand if removes the `before` you would have yo put it in every `it` before the `expect`, i think this is just a way to not repeat it every time in the `it`